openpype/hosts/maya/plugins/publish/validate_path_attributes.py

```diff
@@ -0,0 +1,50 @@
+import os
```

Comment:
Think we should rename the file to `validate_plugin_path_attributes.py` to reflect the class name.

Reply:
I got the name changed. thanks!